SoftLayer/CLI/vlan/cancel.py

```diff
@@ -0,0 +1,30 @@
+"""Cancel Network Vlan."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
+from SoftLayer.CLI import formatting
+from SoftLayer.managers.billing import BillingManager
+
+
+@click.command()
+@click.argument('identifier')
+@environment.pass_env
+def cli(env, identifier):
+    """Cancel network vlan."""
+
+    mgr = SoftLayer.NetworkManager(env.client)
+    billing = BillingManager(env.client)
+    if not (env.skip_confirmations or formatting.no_going_back(identifier)):
+        raise exceptions.CLIAbort('Aborted')
+
```

Comment:
adding a resolve vlan name as `helpers.resolve_id(manager.resolve_ids, volume_id, 'Network Vlan')` would be nice

Reply:
The problem with a resolver here is that vlan Number and vlan Name are both not unique fields, so you might accidently cancel a vlan you didn't mean to. Lets just stick with vlan ID here.